forth.fth

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env gforth
```

Comment:
There needs to be a space between the `#!` and the `/` or the Forth interpreter won't accept it